crates/runtime/src/accelerated_table/refresh_task.rs

```diff
@@ -187,55 +186,65 @@ impl RefreshTask {
         })
     }
 
-    async fn run_once(&self) -> Result<(), RetryError<super::Error>> {
-        self.mark_dataset_status(status::ComponentStatus::Refreshing)
-            .await;
-
-        let dataset_name = self.dataset_name.clone();
-
-        let mode = self.refresh.read().await.mode;
+    async fn run_once(&self, request: &Refresh) -> Result<(), RetryError<super::Error>> {
```

Comment:
Might be better to keep `refresh` name?

Reply:
I will. Initially I had it as `request` when the default `Refresh` was still in `RefreshTask.refresh`.